draft-ietf-quic-http.md

```diff
@@ -1268,8 +1268,9 @@ express the cause of a connection or stream error.
 
 ## HTTP/QUIC Error Codes {#http-error-codes}
 
-The following error codes are defined for use in QUIC RST_STREAM, STOP_SENDING,
-and APPLICATION_CLOSE frames when using HTTP/QUIC.
+The following error codes are defined for use in QUIC RST_STREAM frames,
+STOP_SENDING frames, and CONNECTION_CLOSE frames with a type of 0x03 when using
+HTTP/QUIC.
```

Comment:
I like the idea of a single close frame, but if it is necessary to refer to frame sub-type by numerical value, I think the idea has gone too far.

Reply:
I tend to agree, I'd remove the type reference.